org/org_user_management.py

```diff
@@ -0,0 +1,165 @@
+                    inactive_users.add(user)
+
+            has_next_page = result["data"]["organization"]["membersWithRole"]["pageInfo"]["hasNextPage"]
+            after_cursor_value = result["data"]["organization"]["membersWithRole"]["pageInfo"]["endCursor"]
+
+        return inactive_users
+
+    def _load_yaml_file(self, path):
+        with open(path, "r") as stream:
+            return yaml.safe_load(stream)
+
+    def _write_yaml_file(self, path, data):
+        with open(path, "w") as f:
+            yaml.dump(data, f)
+
+    def delete_inactive_contributors(self, users_to_delete):
+        path = f"{_SCRIPT_PATH}/contributors.yml"
+        contributors_yaml = self._load_yaml_file(path)
+        contributors_yaml["contributors"] = [c for c in contributors_yaml["contributors"] if c not in users_to_delete]
+        self._write_yaml_file(path, contributors_yaml)
+
+    def get_inactive_users_msg(self, users_to_delete, tagusers):
+        rfc = (
+            "https://github.com/cloudfoundry/community/blob/main/toc/rfc/"
+            "rfc-0025-define-criteria-and-removal-process-for-inactive-members.md"
+        )
+        rfc_revocation_rules = (
+            "https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0025-define-"
+            "criteria-and-removal-process-for-inactive-members.md#remove-the-membership-to-the-cloud-foundry-github-organization"
+        )
+        user_tagging_prefix = "@" if tagusers else ""
+        users_as_list = "\n".join(str(user_tagging_prefix + s) for s in users_to_delete)
+        return (
+            f"According to the rolues for inactivity defined in [RFC-0025]({rfc}) following users will be deleted:\n"
+            f"{users_as_list}\nAccording to the [revocation policy in the RFC]({rfc_revocation_rules}), users have"
+            " two weeks to refute this revocation, if they wish."
+        )
+
+    @staticmethod
+    def _get_bool_env_var(env_var_name, default):
+        return os.getenv(env_var_name, default).lower() == "true"
+
+
+if __name__ == "__main__":
+    one_year_back = (datetime.datetime.now() - datetime.timedelta(days=365)).strftime("%Y-%m-%dT%H:%M:%SZ")
+
+    parser = argparse.ArgumentParser(description="Cloud Foundry Org Inactive User Handler")
+    parser.add_argument("-goid", "--githuborgid", default="O_kgDOAAl8sg", help="Cloud Foundry Github org ID")
+    parser.add_argument("-go", "--githuborg", default="cloudfoundry", help="Cloud Foundry Github org name")
+    parser.add_argument("-sd", "--sincedate", default=one_year_back, help="Since when to analyze in format 'Y-m-dTH:M:SZ'")
+    parser.add_argument(
+        "-gt", "--githubtoken", default=os.environ.get("GH_TOKEN"), help="Github API access token. Supported also as env var 'GH_TOKEN'"
+    )
+    parser.add_argument(
+        "-dr",
+        "--dryrun",
+        default=InactiveUserHandler._get_bool_env_var("INACTIVE_USER_MANAGEMENT_DRY_RUN", "False"),
```

Comment:
I'm afraid parsing of a boolean flag won't work. --dryrun 0 or false or False would lead to args.dryrun is true-ish (non-emoty string). type=bool won't work either.

https://stackoverflow.com/questions/60999816/argparse-not-parsing-boolean-arguments provides a solution.

Reply:
This should be addressed with the latest version